src/gui/torrentcontentmodel.cpp

```diff
@@ -351,6 +351,12 @@ QVariant TorrentContentModel::data(const QModelIndex &index, int role) const
                 return Qt::PartiallyChecked;
             return Qt::Checked;
         }
+    case Qt::TextAlignmentRole: {
+        if ((index.column() == TorrentContentModelItem::COL_SIZE)
+            || (index.column() == TorrentContentModelItem::COL_REMAINING))
+            return QVariant {Qt::AlignRight | Qt::AlignVCenter};
```

Comment:
maybe just `return {Qt::AlignRight | Qt::AlignVCenter};`?

Reply:
it's not implicitly convertible.